src/include/splash/basetypes/ColTypeBool.hpp

```diff
@@ -25,6 +26,10 @@
 #define	COLTYPEBOOL_H
 
 #include "splash/CollectionType.hpp"
+#include "splash/basetypes/ColTypeUnknown.hpp"
```

Comment:
what is the ColTypeUnknown include for? (note that there are multiple occurences in other type headers)